code/modules/clothing/head/jobs.dm

```diff
@@ -146,19 +145,63 @@
 	dog_fashion = /datum/dog_fashion/head/warden
 
 /obj/item/clothing/head/warden/drill
-	name = "warden's drill hat"
-	desc = "A special armored campaign hat with the security insignia emblazoned on it. Uses reinforced fabric to offer sufficient protection. Has the letters 'FMJ' enscribed on its side."
+	name = "warden's campaign hat"
+	desc = "A special armored campaign hat with the security insignia emblazoned on it. Uses reinforced fabric to offer sufficient protection."
 	icon_state = "wardendrill"
 	item_state = "wardendrill"
 	dog_fashion = null
-
-/obj/item/clothing/head/warden/drill/equipped(mob/living/carbon/human/user, slot)
-	..()
-	if(slot == SLOT_HEAD)
-		user.dna.add_mutation(YELLING)
-
-/obj/item/clothing/head/warden/drill/dropped(mob/living/carbon/human/user)
-		user.dna.remove_mutation(YELLING)
+	var/anger = 1

```

Comment:
Formalizing this: Please Use Defines

Reply:
what's the difference between using vars and defines for this case in particular? sounds unneeded for something like this